ckanext/versioning/plugin.py

```diff
@@ -13,7 +13,7 @@
 from ckanext.versioning.logic import action, auth, helpers, uploader
 from ckanext.versioning.model import tables_exist
 
-UPLOAD_TS_FIELD = uploader.UPLOAD_TS_FIELD
+UPLOAD_TS_FIELD = 'versions_upload_timestamp'
```

Comment:
This can be removed, along with any code setting it and adding it to the schema in `plugin.py`, because it is no longer needed. 

We added `UPLOAD_TS_FIELD` because in the previous implementation, the upload timestamp was used as part of the in-storage file name to distinguish different revisions of the same file. This is no longer the model and in any case is handled elsewhere and not in versioning.